TWEET.md

```diff
@@ -0,0 +1,6 @@
+🆕 Release $version is out! Thanks to $contributor for the PR 👏
```

Comment:
whats this?

Reply:
isn't it generated automatically?

Reply:
it's the text sent here: https://x.com/strawberry_gql/